docs/topics/testing/tools.txt

```diff
@@ -1381,6 +1381,24 @@ your test suite.
     ``errors`` is an error string, or a list of error strings, that are
     expected as a result of form validation.
 
+.. method:: SimpleTestCase.assertFormValid(form, msg=None)
+
+    Asserts that a form is valid. if form is not valid. Displays clean output with
```

Comment:
Please use periods after all sentences and add `.. versionadded:: 1.10` annotations as described in the [patch review checklist](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/submitting-patches/#patch-review-checklist). Also a mention in the 1.10 release notes is needed. Be sure to wrap all documentation at 79 characters.